gravity-forms/gw-delay-feed-till-user-registration.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Delay a feed until User Registration is Complete.
+ *
+ * Instruction Video: https://www.loom.com/share/176456848c614ab6a3792ca1f9ed9c14
+ *
+ * Delay a specified feed until user registration is complete.
+ * This is useful when you have a feed that needs to be processed after user registration is complete.
+ */
+class GW_Delayed_Feed_Processing {
```

Comment:
I'd update this to more closely match the filename.

Reply:
Done